R/mrmash_wrapper.R

```diff
@@ -301,6 +325,30 @@ compute_w0 <- function(Bhat, ncomps) {
 }
 
 
+# #' @importFrom foreach foreach dopar
+# #'
+# ###Filter data-driven matrices 
+# filter_datadriven_mats <- function(Y, datadriven_mats, sample_partition=NULL){
+    
+#     tissues_to_keep <- colnames(Y)
+    
+#     if (!is.null(sample_partition)){
+#         folds <- sort(unique(sample_partition$fold))
+#         datadriven_mats_filt <- foreach(i = seq_along(folds)) %dopar% {
+#             myfold <- paste0("fold_", i)
+#             # Apply the function to each list within the sub-list of prior_matrices_cv
+#             lapply(datadriven_mats[[myfold]], function(x,to_keep) {x[to_keep, to_keep]}, tissues_to_keep)
+#         }
+#         names(datadriven_mats_filt) <- names(datadriven_mats)
+            
+#     } else {
+#           # if no cross validation 
+#         datadriven_mats_filt <- lapply(datadriven_mats, function(x, to_keep){x[to_keep, to_keep]}, tissues_to_keep)
+#     }
+#     return(datadriven_mats_filt)
+    
+# }
+
```

Comment:
@Chunmingl are these commented out codes still useful? If you are not sure about it please discuss with us on slack. if you are sure they are not useful please remove them.

Reply:
It has been cleared and updated